graph/coloring/doc.go

```diff
@@ -0,0 +1,5 @@
+// Package coloring provides implementation of different graph coloring
+// algorithms, e.g. coloring using BFS, using Backtracking, using greedy
+// approach.
+// Author(s): [Shivam](https://github.com/Shivam010)
+package coloring
```

Comment:
I already added that here. 
Are referring to something else?

Reply:
Oh no that's it. Sorry missed that. Thought there was a newline here 😅